src/services/kratos/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./auth-phone-no-password"
+export * from "./tests-but-not-prod"
```

Comment:
This should not be exported as they are not (yet) part of the public interface (exposed via `index`). I suggest the tests that need them to depend directly on the sub-module `@services/kratos/tests-but-not-prod`.